specification/containerservices/resource-manager/Microsoft.ContainerService/2017-08-31/examples/ManagedClustersAccessProfilesGet.json

```diff
@@ -0,0 +1,22 @@
+{
+  "parameters": {
+    "api-version": "2017-08-31",
+    "subscriptionId": "subid1",
+    "resourceGroupName": "rg1",
+    "resourceName": "clustername1",
+    "accessProfileRoleName": "clusterUser"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid1/resourcegroups/rg1/providers/Microsoft.ContainerService/managedClusters/clustername1/accessProfiles/clusterUser",
+        "location": "location1",
+        "name": "clusterUser",
+        "AccessProfile": {
```

Comment:
`AccessProfile` is the name of the model, I think this should just be `properties`